packages/server/src/migrations/index.ts

```diff
@@ -73,6 +74,14 @@ export const buildMigrations = () => {
         })
         break
       }
+      case MigrationName.TABLE_SETTINGS: {
+        serverMigrations.push({
+          ...definition,
+          appOpts: { all: true },
```

Comment:
It won't have an affect since it's only adding new values, as opposed to changing or deleting existing values, but yea there's no reason to run it on prod apps since it's impossible for prod app client libraries to update without first being published again from the dev app. So I guess we can filter them out to save time.